app/views/ladder/LadderView.coffee

```diff
@@ -41,11 +41,19 @@ module.exports = class LadderView extends RootView
 
   constructor: (options, @levelID, @leagueType, @leagueID) ->
     super(options)
+
+  initialize: ->
```

Comment:
It should work to give `initialize` the same arguments as `constructor`, and remove `constructor`.